### ISSUE TITLE
LAMMPS trajectories should set atoms.names equal to element names

### ISSUE THREAD

**Issue by orioncohen** [MDAnalysis/mdanalysis#3394]
## Is your feature request related to a problem? ##
Whenever I use MDAnalysis to handle LAMMPS trajectories, I have to overwrite the atom names and set them equal to the elements. Otherwise, when writing .xyz files, the elements are not properly assigned.

## Describe the solution you'd like ##
I would like the LAMMPS trajectory to automatically guess elements based on mass and assign them to atom.names. (and fail gracefully)

## Describe alternatives you've considered ##
Alternatively, the XYZ writer could be modified to automatically assign element types if masses are present but names are missing.

**Comment by IAlibay** [comment]
As far as I know the XYZWriter now writes out elements from `atoms.elements` instead of using `atoms.names` if the former is available.

The best thing here would probably be to make it so that whatever favourite LAMMPS format feeds correct elements to `atoms.elements` instead?

Is there another more stable property we could use for element getting here? There have been many decisions over the years about "guessing elements by default", and the general view is that we want to avoid doing that as much as possible. Whilst masses can be a reasonable direct translation to elements in normal cases, in _a lot_ of edge cases they can be problematic (CG, hydrogen mass repartitioning, united atom force fields, etc...).

Tagging @lilyminium who has been part of a lot of these conversations too.

**Comment by orioncohen** [comment]
Ahh ok, that is probably the case. I haven't checked out the code for the XYZWriter. Will take a look.

Got it, those are tricky. In that case, maybe assign elements to atoms if and only if all atom masses can be matched with an element?

 Or, add an optional to guess elements as a keyword argument? Currently the XYZWriter basically fails for unmodified LAMMPS trajectories, which is frustrating.

**Comment by lilyminium** [comment]
Yeah, we have a lot of issues about guessing. I'm guessing LAMMPS trajectories don't have a trivial way to obtain elements, like giving particles atomic numbers? What does LAMMPS do about types, are they named in a way that makes it easy to guess elements? I notice the topology parser has a `massdict` argument for assigning masses via types, which basically sounds like the inverse of what you want to do.

**Comment by orioncohen** [comment]
It may be possible to set atom types in LAMMPS that are compatible with the MDAnalysis reader (I'm no LAMMPS super-user) but in all the LAMMPS simulations I have done, LAMMPS just assigns the types as integers from 1...N. So sadly, pretty useless.

Regardless, it would be convenient to have an option to guess from masses if the atomic numbers aren't specified in the LAMMPS input file. In #3395, @IAlibay suggested adding it as an optional argument to TopologyReaderBase or Universe, which makes sense to me.
